.travis.yml

```diff
@@ -18,11 +18,11 @@ install:
     - npm run bootstrap
 script:
     - echo -e "\x1b\x5b35;1m*** Starting tests...\x1b\x5b0m"
-    - npm test -- -- -- --runInBand --coverage
+    - npm test -- -- --runInBand --coverage
```

Comment:
Would you be more specific for this change?

Reply:
I cannot found about `--` argument on lerna docs(https://lerna.js.org/docs/introduction)
But these changes can fix the travis.